template.xml

```diff
@@ -147,6 +108,26 @@
         </list>
        </t>
    </section>
+
+  <section title="Contributors">
+  <t>The GeoJSON draft is primarily the work of the following individuals.</t>
+  
+  <t>Howard Butler<vspace/>Hobu Inc.<vspace blankLines="1"/>Email: howard@hobu.co<vspace blankLines="1"/></t>
```

Comment:
If you wrap this list in `<list style="none">` and `</list>` this might render a little more.  Then you can use a colon in the lead-in sentence.